monkestation/code/modules/divine_warning/code/effects.dm

```diff
@@ -0,0 +1,76 @@
+#define HEAVEN_TIMETOGO (1<<0)
+#define HEAVEN_TIMETOGO_PLAYED (1<<1)
+
+/atom/movable/screen/fullscreen/divine
+	icon = 'monkestation/code/modules/divine_warning/icons/divine_warning.dmi'
+	icon_state = "he_waits_for_you"
+	layer = EMISSIVE_LAYER_UNBLOCKABLE
+	alpha = 120
+
+/mob/living/proc/flash_divine_overlay(alpha = 120, soundvolume = 80, time = 2 SECONDS, force = FALSE)
+	if(client?.prefs?.read_preference(/datum/preference/toggle/darkened_flash))
+		clear_fullscreen("divine", time)
+		return
+	if (COOLDOWN_FINISHED(src, divine_cooldown) || force)
+		soundvolume *= 0.8
+		SEND_SOUND(src, sound('monkestation/code/modules/divine_warning/sounds/divine.ogg', volume = soundvolume))
+		overlay_fullscreen("divine", /atom/movable/screen/fullscreen/divine, 1, alpha)
+		COOLDOWN_START(src, divine_cooldown, 2.5 SECONDS)
+	clear_fullscreen("divine", time)
+
+/mob/living/death(gibbed)
+	. = ..()
+	if (!client || !src.mind || stat == DEAD) return
```

Comment:
```suggestion
	if (QDELETED(client) || stat == DEAD)
		return
```
no need to check mind tbh - you only use mind once, to check `holy_role`, and you can just replace that with a `?.` operator